klippy/extras/manual_stepper.py

```diff
@@ -109,7 +109,7 @@ def flush_step_generation(self):
         self.sync_print_time()
     def get_position(self):
         return [self.rail.get_commanded_position(), 0., 0., 0.]
-    def set_position(self, newpos, homing_axes=()):
+    def set_position(self, newpos):
```

Comment:
This change doesn't look correct to me.

-Kevin

Reply:
I was not sure whether this `set_position` complies to the same interface as the toolhead. From what I can see, it's essentially implementing the same interface as [`PrinterRail.set_position`](https://github.com/Klipper3d/klipper/blob/8a3d2afd796414d64b995ac753148484b77198dd/klippy/stepper.py#L419), and thus shouldn't have the homing info.

Reply:
No - the manual_stepper class is passed to homing.py as an emulated "toolhead" and thus it should follow the toolhead `set_position()` signature.

-Kevin

Reply:
All clear, fixed!